api/models/restaurantDao.js

```diff
@@ -9,14 +9,70 @@ const districts = async() => {
         name
       FROM districts
       `
-    )
+    );
   } catch {
-    const error = new Error('DATASOURCE_ERROR');
+    const error = new Error("DATASOURCE_ERROR");
     error.statusCode = 400;
     throw error;
   }
 };
+const getRestaurantList = async (conditionQuery) => {
+  try {
+    let conditionWhereQuery = [];
+    const {
+      district,
+      Date,
```

Comment:
Date는 JS이미 지정된 class가 있을겁니다. 따라서 다음과 같이 변경해 주세요.
```suggestion
      date,
```